src/nri/sampler.go

```diff
@@ -53,7 +48,7 @@ func NewSampler(fetcher raw.Fetcher, docker raw.DockerClient, config config.Argu
 	store, err := persist.NewFileStore(
 		persist.TmpPath(config.TempDir, "container_cpus"),
 		log.NewStdErr(true),
-		cacheTTL)
+		config.CacheTTL)
```

Comment:
I guess we need to bump the sdk when https://github.com/newrelic/infra-integrations-sdk/pull/316 is released